tests/regressionTest.m

```diff
@@ -38,95 +38,105 @@ function killPlots (~)
     
     methods(TestClassSetup)
         
-        function runRegTest(testCase)            
+        function runBEMIO(testCase)
+            cd(fullfile(testCase.testDir,...
+                        '..',...
+                        'examples',...
+                        'RM3',...
+                        'hydroData'));
+            bemio
+            cd(testCase.testDir);
+        end
+
```

Comment:
I wonder if this test is better suited for `bemioTest.m`, instead of the RM3 regression test. If there is an issue with BEMIO, it seems like we should test it in `bemioTest.m`. That also allows us to compare across BEM solvers, and to a prior solution. 

Right now, we run the sphere cases in all BEM solvers, but it doesn't look like we compare the solutions. The sphere was chosen because it's a simple/fast case, but we could run the RM3 and compare the results to the uploaded h5, in the RM3 example. 

@akeeste what do you think about this suggestion?